File: api/DTOs/Finance/PendingDebitApprovalDto.cs
namespace api.DTOs.Finance
{
    public class PendingDebitApprovalDto
    {
        public bool Selected { get; set; }
        public int Id { get; set; }
        public int VoucherEntryId {get; set;}
        public int VoucherNo {get; set;}
        public DateOnly VoucherDated {get; set;}
        public int DrAccountId { get; set; }
        public string DrAccountName { get; set; }
        public long DrAmount {get; set;}
        public bool DrEntryApproved {get; set;}
        public string DrEntryApprovedByAppUsername {get; set;}
        public DateTime DrEntryApprovedOn {get; set;}
    }
}

File: client/src/app/_dtos/finance/pendingDebitApprovalDto.ts
export interface IPendingDebitApprovalDto
{
	id: number;
	voucherEntryId: number;
	voucherNo: number;
	voucherDated: Date;
	drAccountId: number;
	drAccountName: string;
	drAmount: number;
	drEntryApproved: boolean;
	drEntryApprovedByUsername: string;
	drEntryApprovedOn: Date;
	selected: boolean;
}

export class PendingDebitApprovalDto implements IPendingDebitApprovalDto
{
	id=0;
	voucherEntryId = 0;
	voucherNo=0;
	voucherDated=new Date();
	drAccountId = 0;
	drAccountName = '';
	drAmount = 0;
	drEntryApproved = false;
	drEntryApprovedByUsername = '';
	drEntryApprovedOn = new Date();
	selected = false;
}